src/com/example/ui.cljs

```diff
@@ -89,7 +91,7 @@
            {:header (comp/get-query Header)}
            {:quiz (comp/get-query Quiz)}]
    :initial-state {:root/user {} :header {} :quiz {}}}
-  (div :.container.mx-auto.text-zinc-800
+  (div :.container.mx-auto.text-gray-100
```

Comment:
I was getting fancy with that zinc color lol

Reply:
I do eventually want to customize the tailwind config with the color pallet I have in #4 but keeping it simple for now